plugin.rb

```diff
@@ -87,4 +87,4 @@ def contains_restricted?(attachment)
         @restricted_file_types = SiteSetting.file_attachment_whispers_file_extensions.split('|')
         @restricted_file_types.any? { |extension| attachment['href'].include?(extension)}
     end
-end
\ No newline at end of file
+end
```

Comment:
looks like your editor touched the whitespace at the end too. just needs to be a separate commit.